prawn-templates.gemspec

```diff
@@ -20,13 +22,11 @@ Gem::Specification.new do |spec|
     'gregory.t.brown@gmail.com', 'brad@bradediger.com', 'dnelson@bluejade.com',
     'greenberg@entryway.net', 'jimmy@deefa.com', 'b.vogel@buddyandselly.com'
   ]
-  spec.add_dependency('pdf-reader', '~> 2.0')
+  spec.add_dependency('pdf-reader', '~> 2.0', '!= 2.9.0', '!= 2.9.1')
   spec.add_dependency('prawn', '~> 2.2')
   spec.add_development_dependency('pdf-inspector', '~> 1.3')
+  spec.add_development_dependency('prawn-dev', '~> 0.3.0')
   spec.add_development_dependency('rspec', '~> 3.0')
```

Comment:
This is a dep of prawn-dev, too.

Reply:
@pointlessone the `rspec` line has been removed and this should be resolved.  Can you please take another look?  Thanks.